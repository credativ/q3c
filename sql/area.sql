select 6*q3c_pixarea(0,30)/pi()/4. between 0.99999 and 1.00001;
select q3c_pixarea(10,31);
select q3c_pixarea(10,0);
select q3c_pixarea(-1,4);
select q3c_pixarea(6917529027641081856,4);

