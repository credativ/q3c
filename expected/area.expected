 ?column? 
----------
 t
(1 row)

ERROR:  Invalid depth. It should be less than 31.
ERROR:  Invalid depth. It should be greater than 0.
ERROR:  Invalid ipix value
ERROR:  Invalid ipix value
